proposals/lip-0002.md

```diff
@@ -27,7 +27,7 @@ As the Lisk protocol evolves and gains popularity, a clear increase in usage and
 We propose to set a maximum block size of 15 KiB (15 * 1024 bytes). This size will maintain the yearly blockchain growth below 50 GiB, even when blocks are always at full capacity, and the active delegates do not miss any blocks. The summary of the numerical implications are as follows:
 
 * Maximum block size: 15 KiB.
-* Blockchain size will grow a **maximum** of 47.3 GiB per year.
+* The block history will grow around 45 GiB per year (approximately 6*60*24*365.25*15/(1024*1024) GiB).
```

Comment:
But "around" instead of "maximum" is rather misleading as we have very few transactions right now. Don't you think?

Reply:
I added "assuming full blocks" to the end of the sentence.